doc/news/_preparation_next_release.md

```diff
@@ -61,6 +61,12 @@ The following section lists news about the [plugins](https://www.libelektra.org/
 
 ### <<Plugin3>>
 
+### TOML
+
+- Improvements to the parser, comment handling and especially quoting of strings. _(Klemens Böswirth)_
```

Comment:
Can you please write a bit more here? At least the fix that `comment/#/space` now contains spaces/tabs instead of a number should be mentioned here explicitly. Please also link to src/plugins/toml/README.md 

Great job! :rocket: